code/modules/mining/coins.dm

```diff
@@ -44,6 +44,28 @@
 	name = "platinum coin"
 	icon_state = "coin_adamantine"
 
+/obj/item/weapon/coin/erp
+	name = "erp token"
+	icon_state = "erp_token"
+	desc = "Put this in her slot to redeem your erp token. Good for 1 deciseconds of erp."
+
+/obj/item/weapon/coin/erp/attack(/mob/living/carbon/human/target, /mob/living/carbon/human/user, var/target_zone)
```

Comment:
Seems pretty arbitrary to limit ERP tokens to being redeemable only human mobs. I know personally myself that in the old system I spent many ERP tokens on player-occupied subhumans such as Dionaea nymphs, Ian, mice, and Tajara.